src/main/java/tools/jackson/databind/introspect/AnnotatedCreatorCollector.java

```diff
@@ -288,7 +289,7 @@ protected AnnotatedConstructor constructNonDefaultConstructor(ClassUtil.Ctor cto
             ClassUtil.Ctor mixin)
     {
         final int paramCount = ctor.getParamCount();
-        if (_intr == null) { // when annotation processing is disabled
+        if (_intr == null || _intr == AnnotationIntrospector.nopInstance()) { // when annotation processing is disabled
```

Comment:
Hmmmh. That seems unfortunate... nop instance is valid introspector. So ideally would not have to do this. But maybe it is necessary.

Reply:
I think this would be fixed by changing to

    if (!_collectAnnotations)

but need to check with other changes. Thanks!